eng/configure-toolset.ps1

```diff
@@ -3,3 +3,8 @@
 # using the globally installed dotnet
 
 $script:useInstalledDotNetCli = $false
+
+# Most of this repo does not yet go through the full Arcade
```

Comment:
It is unclear if we will ever want to use the global nuget cache. There seem to be too many potential problems that could arise from it in our CI to warrant using it.

Reply:
The Arcade scripts are set up such that if the ci flag is specified, they will not use the global nuget cache, so it would only be for local dev builds. However, I would definitely get (/ agree with) the argument that we would want our local builds to be as close to ci/official as possible. I'll update the comment to just say that we want the local packages directory without referring to other systems / potential future plans.